src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -495,7 +490,7 @@ void BaseQNAcceleration::splitCouplingData(
     int   size       = cplData[id]->values().size();
     auto &valuesPart = cplData[id]->values();
     //Eigen::VectorXd& oldValuesPart = cplData[id]->oldValues.col(0);
-    cplData[id]->oldValues.col(0) = _oldValues.segment(offset, size); /// @todo: check if this is correct
+    cplData[id]->writeLastIteration(_oldValues.segment(offset, size)); /// @todo: check if this is correct
```

Comment:
@uekerman This is now actually the only place, where `CouplingData::writeLastIteration` is called. Everywhere else we only use `CouplingData::storeIteration` to shift the old data to the buffer.

Reply:
We removed this part, since it is not needed.